package.json

```diff
@@ -68,6 +68,8 @@
     "babel-register": "^6.6.0",
     "chai": "^3.4.1",
     "del": "^2.2.0",
+    "electron-dl": "^1.8.0",
```

Comment:
@nikvdp mmmmmh, unless I'm missing something (I've been helping maintenance for very little time), please revert adding these dependencies. They are necessary only for in-app purposes, so we depend on them through separate [app/package.json](https://github.com/jiahaog/nativefier/blob/development/app/package.json#L6-L11), and installing them for development is documented in [the dev guide](https://github.com/jiahaog/nativefier/blob/development/docs/development.md).